### ISSUE TITLE
DeprecationWarning: rebin_test.py

### ISSUE THREAD

**Issue by samtygier-stfc** [mantidproject/mantidimaging#970]
### Summary

```
../../miniconda3/envs/midev-py38/lib/python3.8/site-packages/skimage/util/dtype.py:226: 1 warning
mantidimaging/core/operations/rebin/test/rebin_test.py: 80 warnings
  /home/sam/miniconda3/envs/midev-py38/lib/python3.8/site-packages/skimage/util/dtype.py:226: DeprecationWarning: Converting `np.inexact` or `np.floating` to a dtype is deprecated. The current result is `float64` which is not strictly correct.
    dtypeobj_out = np.dtype(dtype)

../../miniconda3/envs/midev-py38/lib/python3.8/site-packages/tomopy/recon/acceleration.py:57
  /home/sam/miniconda3/envs/midev-py38/lib/python3.8/site-packages/tomopy/recon/acceleration.py:57: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    import imp
```

### Steps To Reproduce

`pytest -vs mantidimaging/core/operations/rebin/test/rebin_test.py`

**Comment by samtygier-stfc** [comment]
skimage part is fixed by an update to 0.18

tomopy part is fixed by an update to 1.10

**Comment by samtygier-stfc** [comment]
Tomopy 1.10 needs cudatoolkit 10.2 or newer, which conflicts with IDAaaS. So will have to wait.
